frontend/src/pages/AlertDetails/AlertHeader/ActionButtons/index.tsx

```diff
@@ -0,0 +1,64 @@
+import './actionButtons.styles.scss';
+
+import { Button, Divider, Dropdown, MenuProps, Tooltip } from 'antd';
+import { Copy, Ellipsis, PenLine, Trash2 } from 'lucide-react';
+import CopyToClipboard from 'periscope/components/CopyToClipboard';
+import React from 'react';
+
+const menu: MenuProps['items'] = [
+	{
+		key: 'rename-rule',
+		label: 'Rename',
+		icon: <PenLine size={16} color="var(--bg-vanilla-400" />,
+		onClick: (): void => {},
+	},
+	{
+		key: 'duplicate-rule',
+		label: 'Duplicate',
+		icon: <Copy size={16} color="var(--bg-vanilla-400" />,
+		onClick: (): void => {},
+	},
+];
+
+const menuStyle: React.CSSProperties = {
+	padding: 0,
+	boxShadow: 'none',
+	fontSize: 14,
+};
+
+function AlertActionButtons(): JSX.Element {
+	return (
+		<div className="alert-action-buttons">
+			<CopyToClipboard textToCopy={window.location.href} />
+
+			<Divider type="vertical" />
+
+			<Dropdown
+				trigger={['click']}
+				menu={{ items: menu }}
+				// eslint-disable-next-line react/no-unstable-nested-components
+				dropdownRender={(menu): JSX.Element => (
+					<div className="dropdown-menu">
+						{React.cloneElement(menu as React.ReactElement, {
+							style: menuStyle,
+						})}
+						<Divider style={{ margin: 0 }} />
+						<Button
+							type="default"
+							icon={<Trash2 size={16} color="var(--bg-cherry-400" />}
+							className="delete-button"
+						>
+							Delete
+						</Button>
+					</div>
+				)}
+			>
+				<Tooltip title="More options">
+					<Ellipsis size={16} color="var(--bg-vanilla-400)" cursor="pointer" />
```

Comment:
we have `Color.BG_VANILLA_400` for usage in ts files. better to use that ?

Reply:
Done